### PR TITLE
SB-300-next-episode

### PR DESCRIPTION
**Pull request by fuss86**

### CODE REVIEW

File: strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexer.java [hunk 1]
```diff
@@ -55,37 +58,43 @@
...
+                                                                                         artifactPath.toAbsolutePath().toFile());
 
-            ArtifactContext artifactContext = artifactContextProducer.getArtifactContext(indexingContext, artifactFile);
+            if (artifactContext == null)
+            {
+                return;
+            }
+            final ArtifactInfo artifactInfo = artifactContext.getArtifactInfo();
 
+            // preserve duplicates
```
Comment (carlspring) [inline strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexer.java]: 👍

**Comment by carlspring** [comment]
Your PR looks good, thanks!

**Comment by carlspring** [comment]
Please, let me know, if I should proceed with the merging.

**Comment by fuss86** [comment]
You can merge it, @carlspring, but we're using our fork of maven-indexer here.

**Comment by carlspring** [comment]
Thanks! :)

**Pull request merged**